python/graphstorm/gconstruct/construct_graph.py

```diff
@@ -87,17 +100,30 @@ def write_data_parquet(data, data_file):
     table = pa.Table.from_arrays(list(arr_dict.values()), names=list(arr_dict.keys()))
     pq.write_table(table, data_file)
 
-def parse_file_format(fmt):
+def parse_file_format(conf, is_node):
     """ Parse the file format blob
 
     Parameters
     ----------
-    fmt : dict
-        Describe the file format.
+    conf : dict
+        Describe the config for the node type or edge type.
+    is_node : bool
+        Whether this is a node config or edge config
+
```

Comment:
Can we have an example of the config here?
```
Example of conf:
{
  "format": {
    "name": "parquet",
  }
  "node_id_col": ID,
  "features": [feat0, feat1, ...],
  "labels": [label0],
}
```

Reply:
The config has been documented in `process_node_data` and `process_edge_data`.